src/chrome/content/rules/ArchiveTeam.org.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+        Invalid certificate:
+                tracker.archiveteam.org:1337
+
+        Redirect to HTTP:
+
+
+        Refused:
+                fileformats.archiveteam.org
+                digitize.archiveteam.org
+                iabak.archiveteam.org
+                justsolve.archiveteam.org
+
+
+        Time out:
+                metadata.archiveteam.org
+
+-->
+<ruleset name="ArchiveTeam.org">
+    <target host="archiveteam.org/"/>
+    <target host="www.archiveteam.org/"/>
+    <target host="tracker.archiveteam.org/"/>
+	<rule from="^http://tracker\.archiveteam\.org(:1337)?/" to="https://tracker.archiveteam.org:1338/"/>
+	<target host="warriorhq.archiveteam.org/"/>
+
+	<securecookie host=".+" name=".+"/>
+
+    <rule from="^http:" to="https:"/>
+</ruleset>
```

Comment:
Just for the sake of good arrangement please put the rules in one place so that it becomes easier to review like this:

```
<securecookie host=".+" name=".+"/>

<rule from="^http://tracker\.archiveteam\.org(:1337)?/" to="https://tracker.archiveteam.org:1338/"/>

<rule from="^http:" to="https:"/>
</ruleset>
```